lib/cloud_controller/runner.rb

```diff
@@ -17,7 +17,7 @@
 
 module VCAP::CloudController
   class Runner
-    attr_reader :config, :config_file, :insert_seed_data, :secrets_file
```

Comment:
Is it possible that this is used by a rake task or pipeline that wasn't loaded into the debride analysis context?

Reply:
There's no reference to it in cloud_controller_ng or capi-release.  [capi-release has references to CCNG's Drain code that debride didn't catch](https://github.com/search?q=repo%3Acloudfoundry%2Fcapi-release%20CloudController%3A%3ADrain&type=code), but as far as I can tell that is the only place CCNG's code is referenced externally (and arguably should be replaced with a rake task)